### ISSUE TITLE
KL

### ISSUE THREAD

**Issue by SaintLogos1234** [vlfeat/autonn#42]
My recently working is using Deeplearning to solve the tracking problem, I need to combine KL divergence with my loss function, what can I do to achieve the KL divergence's forward and backward propagation by autonn.

**Comment by damdaepark** [comment]
You just need to write down a set of elementary operations that is equal to KL divergence.
For the elementary operations: [Check this out](https://stackoverflow.com/questions/41863814/kl-divergence-in-tensorflow)
Then AutoNN will automatically calculate the backward path for you.

**Comment by SaintLogos1234** [comment]
Thank you for your reply，I will try  do it

**Comment by jotaf98** [comment]
Thanks for responding @damdaepark !
